osu.Game/Screens/Multi/Lounge/LoungeSubScreen.cs

```diff
@@ -62,7 +62,18 @@ private void load()
                                     RelativeSizeAxes = Axes.Both,
                                     ScrollbarOverlapsContent = false,
                                     Padding = new MarginPadding(10),
-                                    Child = roomsContainer = new RoomsContainer { JoinRequested = joinRequested }
+                                    Child = roomsContainer = new RoomsContainer
+                                    {
+                                        JoinRequested = joinRequested,
+                                        DuplicateRoom = room =>
+                                        {
+                                            Room newRoom = new Room();
+                                            newRoom.CopyFrom(room, true);
+                                            newRoom.Name.Value = $"Copy of {room.Name.Value}";
```

Comment:
we might want this to be a bit smarter, to avoid a *Copy of Copy of Copy of (...) __[actual room name]__* sort of situation